### PR TITLE
Visitor can view articles sorted by category

### PR DESCRIPTION
**Pull request by arunbhalli**
PT: https://www.pivotaltracker.com/story/show/176031091
### User Story
```
As a visitor
In order to find a certain type of article I want to read
I would like to be able to view articles by category
```
### Changes proposed in this pull request: 
- Journalist can see the articles in different categories
- Route system for `: category`
## What I have learned working on this feature:
- How to use redux
- Deeper knowledge of hooks
- Styling better

### Video of the site
https://recordit.co/PM6wr3sVQp

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ import { Provider } from 'react-redux';
 import store from './state/store/configureStore';
 import { BrowserRouter as Router } from 'react-router-dom';
 
-axios.defaults.baseURL = 'https://fake-newzzzzz.herokuapp.com/api';
+// axios.defaults.baseURL = 'https://fake-newzzzzz.herokuapp.com/api';
+axios.defaults.baseURL = 'http://localhost:3000/api';
```
Comment (LucasKnudsen) [inline src/index.js]: Change this back bois who tf did this

File: cypress/integration/userCanNavigateAndSeeCategoryPage.feature.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  describe('unsuccessfully There is no article in this category', () => {
+    beforeEach(() => {
+      cy.intercept(
+        'GET',
+        'https://fake-newzzzzz.herokuapp.com/api/articles/**',
+        {
+          articles: [],
+        }
+      );
```
Comment (LucasKnudsen) [inline cypress/integration/userCanNavigateAndSeeCategoryPage.feature.js]: What about a test for amount of articles expected

File: src/index.js (hunk 1 above)
Comment (christerforsgren91) [reply src/index.js]: its fixed now!

File: cypress/integration/userCanNavigateAndSeeCategoryPage.feature.js (hunk 2 above)
Comment (christerforsgren91) [reply cypress/integration/userCanNavigateAndSeeCategoryPage.feature.js]: Good idea, we added it now! 👍

**Review by LucasKnudsen — APPROVED** [review]

File: src/components/Category.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { useParams } from 'react-router';
+import { useSelector } from 'react-redux';
+import CategoryMenu from './layout/CategoryMenu';
+
+const Category = () => {
+  const { articles } = useSelector((state) => state);
+  const { category } = useParams();
+
+  useEffect(() => {
+    // eslint-disable-next-line
```
Comment (offcast-dev) [inline src/components/Category.jsx:13]: Do you need this comment?

File: src/index.js [hunk 4]
```diff
@@ -10,6 +10,7 @@ import store from './state/store/configureStore';
 import { BrowserRouter as Router } from 'react-router-dom';
 
 axios.defaults.baseURL = 'https://fake-newzzzzz.herokuapp.com/api';
+// axios.defaults.baseURL = 'http://localhost:3000/api';
```
Comment (DavDan1) [inline src/index.js]: Please remove the comment :)

File: src/style.css [hunk 5]
```diff
@@ -29,17 +31,16 @@ body {
...
 }
 
 #footer {
-  position: flex;
   bottom: 0;
   margin-top: 1rem;
   padding: 1rem;
   background-color: rgb(19, 18, 17);
-  width: 100vw;
+  width: 100%;
```
Comment (DavDan1) [inline src/style.css:43]: wasn't vw = 100% ?

**Review by DavDan1 — APPROVED** [review]

File: src/style.css (hunk 5 above)
Comment (LucasKnudsen) [reply src/style.css:43]: yee 🦐

File: src/components/Category.jsx (hunk 3 above)
Comment (LucasKnudsen) [reply src/components/Category.jsx:13]: Yep 😢

File: src/index.js (hunk 4 above)
Comment (lukeperrin10) [reply src/index.js]: Removed :)

**Pull request merged**